contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "ethereum-waffle": "^3.2.0",
     "ethers": "^5.0.17",
     "fs-extra": "^9.0.1",
-    "hardhat": "^2.6.0",
+    "hardhat": "^2.6.8",
```

Comment:
Weird issue, but block explorer links seem to be malformed, with an `https//` (missing colon).

```
https://https//polygonscan.com/tx/0x0ed14cd2610a764d9728492b55981af68e0792ef695e7f4305b6daf37cd99398
```

This is happening with blocknative links and also on the receive screen to historical txs.

Reply:
broke earlier in this PR in 1203fca, just fixed in 8efe97a